.github/workflows/pdk-basic.yml

```diff
@@ -0,0 +1,51 @@
+# This is a basic workflow to help you get started with Actions
+
+name: main
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push 
+  push:
```

Comment:
Since this is a reusable workflow, I'm not sure you can have push here. I think that needs to happen in the workflow that refers to this.

Reply:
That would explain a few things.  Just noticed workflows can be referenced.  Updated code.